--- jbigkit/libjbig/Makefile	Fri Aug 10 04:27:28 2001
+++ jbigkit-mod/libjbig/Makefile	Fri Aug 10 04:26:27 2001
@@ -14,10 +14,10 @@
 	jbig_tab.o
 
 %.lo: %.c
-	libtool $(CC) $(CFLAGS) -c $*.c
+	libtool --tag CXX $(CC) $(CFLAGS) -c $*.c
 
 libjbig.la: jbig.lo jbig_tab.lo
-	libtool gcc -o libjbig.la jbig.lo jbig_tab.lo -rpath /usr/lib
+	libtool --tag CXX gcc -o libjbig.la jbig.lo jbig_tab.lo -rpath /usr/lib
 
 jbig.lo: jbig.c jbig.h
 
